checker/tests/initialization/fbc/FieldSuppressWarnings.java

```diff
@@ -0,0 +1,24 @@
+import org.checkerframework.checker.nullness.qual.*;
+
+import java.util.*;
+
+//:: error: (initialization.fields.uninitialized)
```

Comment:
I don't think this test case really tells us whether this feature works.
There is an expected error on the class, presumably because of field notInitialized. However, any of the other fields might still cause the warning.
I would split this class into 5 separate classes to make sure the feature actually works.